HomePage.cs

```diff
@@ -18,28 +18,25 @@ public void startBrowser()
         {
             driver = new ChromeDriver();
             driver.Manage().Window.Maximize();
+            driver.Url = "https://www.saucedemo.com/";
+            System.Threading.Thread.Sleep(2000);
```

Comment:
what would you do if you need the run the tests in the environment which is much slower?